t/22-dashboard.t

```diff
@@ -536,17 +536,4 @@ subtest 'job parent groups with multiple version and builds' => sub {
 
 };
 
-subtest 'extra plugin links' => sub {
```

Comment:
Don't delete the whole subtest. We still want to test the "plugin_links" feature in general.